src/BenchmarkDotNet/Extensions/ProcessExtensions.cs

```diff
@@ -132,6 +134,17 @@ internal static void SetEnvironmentVariables(this ProcessStartInfo start, Benchm
             if (!benchmarkCase.Job.HasValue(EnvironmentMode.EnvironmentVariablesCharacteristic))
                 return;
 
+            if (benchmarkCase.Job.Infrastructure.Toolchain is MonoAotLLVMToolChain)
+            {
+                MonoAotLLVMRuntime aotruntime = (MonoAotLLVMRuntime)benchmarkCase.GetRuntime();
+
+                if (aotruntime.AOTCompilerMode == MonoAotCompilerMode.llvm)
+                {
+                    start.EnvironmentVariables["MONO_ENV_OPTIONS"] = "--llvm,--full-aot";
```

Comment:
You only need `--full-aot` here.

Reply:
yeah and `,` wouldn't work here since arguments are space separated